windows-server-container-tools/Debug-ContainerHost/Debug-ContainerHost.ps1

```diff
@@ -22,12 +22,15 @@ Describe "Windows Version and Prerequisites" {
 
 Describe "Docker is installed" {
     
-    $service = Get-Service | Where-Object {($_.Name -eq "Docker") -or ($_.Name -eq "com.Docker.Service")}
+    $services = Get-Service | Where-Object {($_.Name -eq "Docker") -or ($_.Name -eq "com.Docker.Service")}
     It "A Docker service is installed - 'Docker' or 'com.Docker.Service' " {
-        $service | Should Not Be Null
+        $services | Should Not Be Null
     }
     It "Service is running" {
-        $service.Status | Should Be Running
+        foreach ($service in $services)
+        {
+           $service.Status | Should Be Running
```

Comment:
Oh, I misread the change. Fix line 27 to | Should Not BeNullOrEmpty then it should be fine.